ArticleTemplates/assets/js/bootstraps/liveblog.js

```diff
@@ -131,19 +131,31 @@ function onLiveMoreClick(liveMoreElem) {
 }
 
 function setupTryLive() {
-    const elems = document.getElementsByClassName('live-promo__button');
-    for (let i = 0; i < elems.length; ++i) {
-        let elem = elems[i];
-        elem.addEventListener('touchstart', () => {
-            elem.classList.add('pressed');
-        });
-        elem.addEventListener('touchend', () => {
-            elem.classList.remove('pressed');
-        });
-        elem.addEventListener('click', () => {
-            signalDevice('try-live');
-        });
-    }
+    const elem = document.getElementsByClassName('block--live-promo')[0];
+    if (elem == null) return;
```

Comment:
```suggestion
    if (elem === undefined) return;
```

Reply:
Sure thing. I general the JS community has settled on double-equal + `null` because it's a catch all for all "nullish" states, and it can be fiddly to make the difference. We may want to consider applying an automatic lint rule based on the direction we'd like (though of course, this codebase's going away with SSR ^_^ )

Reply:
Ah actually `querySelector` returns `null`, so I'll go back to `null` ;)

Reply:
Well at least in automated tests